git_info

```diff
@@ -0,0 +1 @@
+{"test": "test"}
```

Comment:
Can you delete this file?  It's not really useful in other repos so no need to duplicate it here.